plugins/modules/redis_info.py

```diff
@@ -193,46 +213,41 @@
 
 REDIS_IMP_ERR = None
 try:
-    from redis import StrictRedis
-    HAS_REDIS_PACKAGE = True
+    import redis
 except ImportError:
     REDIS_IMP_ERR = traceback.format_exc()
-    HAS_REDIS_PACKAGE = False
+    redis_found = False
+else:
+    redis_found = True
 
-from ansible.module_utils.basic import AnsibleModule, missing_required_lib
+from ansible.module_utils.basic import AnsibleModule
 from ansible.module_utils.common.text.converters import to_native
-
-
-def redis_client(**client_params):
-    return StrictRedis(**client_params)
```

Comment:
This helper function was likely needed by the unit tests, which now fail. You need to fix them.

Reply:
tests are fixed in last commit